lib/rsgem/version.rb

```diff
@@ -3,6 +3,6 @@
 module RSGem
   MAJOR = 0
   MINOR = 4
-  PATCH = 0
+  PATCH = 1
```

Comment:
dropping support to older ruby versions is a patch change? 🤔

Reply:
lol good catch, this change shouln't be here. 

I usually merge changes and after that I prepare a version change commit